draft/2022-08-10-this-week-in-rust.md

```diff
@@ -86,36 +86,68 @@ Rusty Events between 2022-08-10 - 2022-09-07 🦀
 
 ### Virtual
 
-* 2022-08-03 | Virtual (Indianapolis, IN, US) | [Indy Rust](https://www.meetup.com/indyrs/)
-    * [**Indy.rs - with Social Distancing**](https://www.meetup.com/indyrs/events/qwtdjsydclbfb/)
-* 2022-08-03 | Virtual (Stuttgart, DE) | [Rust Community Stuttgart](https://www.meetup.com/Rust-Community-Stuttgart/)
-    * [**Rust-Meetup**](https://www.meetup.com/rust-community-stuttgart/events/dvvtvsydclbfb/)
-* 2022-08-05 | Virtual + Portland, OR, US | [RustConf](https://rustconf.com/)
-    * [**RustConf 2022**](https://rustconf.com/)
-* 2022-08-09 | Virtual (Dallas, TX, US) | [Dallas Rust](https://www.meetup.com/Dallas-Rust/)
-    * [**Second Tuesday**](https://www.meetup.com/dallas-rust/events/vndgwsydclbmb/)
-* 2022-08-09 | Virtual (Myrtle Point, OR, US) | [#EveryoneCanContribute Cafe](https://www.meetup.com/everyonecancontribute-cafe/)
-    * [**Summer Chill & Learn: including OpenTelemetry getting started with Rust**](https://www.meetup.com/everyonecancontribute-cafe/events/286609523/)
 * 2022-08-10 | Virtual (Boulder, CO, US) | [Boulder Elixir and Rust](https://www.meetup.com/boulder-elixir-rust/)
     * [**Monthly Meetup**](https://www.meetup.com/boulder-elixir-rust/events/zvxcsrydclbnb/)
+* 2022-08-10 | Virtual (Brasília, BR) | [Turing Community](https://www.meetup.com/turing-community/) + [Turing Community LatAm](https://www.meetup.com/turing-community/members/?op=leaders)
+    * [**Coding Dojo #2: Rust**](https://www.meetup.com/turing-community/events/287559034/)
+* 2022-08-10 | Virtual (Cardiff, UK) | [Rust and C++ Cardiff](https://www.meetup.com/rust-and-c-plus-plus-in-cardiff/)
+    * [**cargo expand --bin writing_proc_macros**](https://www.meetup.com/rust-and-c-plus-plus-in-cardiff/events/287450224/)
+* 2022-08-10 | Virtual (Hoboken, NJ, US) | [Neighborhood Math Club](https://www.meetup.com/neighborhood-math-club/)
+    * [**The Early Rustacean Gets The Worm!**](https://www.meetup.com/neighborhood-math-club/events/287468144/)
```

Comment:
```suggestion
    * [**The Early Rustacean Gets The Worm!**](https://www.meetup.com/neighborhood-math-club/events/kpgpssydclbnb/)
```
Link currently hits the event on the 10th — re-routing

Reply:
TY very much for the catch and the proposed fix